patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -55,7 +55,25 @@
  
          if (this.field_70170_p.func_180495_p(p_180429_1_.func_177984_a()).func_177230_c() == Blocks.field_150431_aC)
          {
-@@ -1277,12 +1289,12 @@
+@@ -1210,7 +1222,8 @@
+         {
+             this.field_70171_ac = false;
+         }
+-
++        
```

Comment:
Remove this whitespace change

Reply:
Okay, done.

Reply:
This whitespace change is still here.
Also please do not ping me in commits, I am getting notified for each one.